### ISSUE TITLE
need sample clients for angular and react

### ISSUE THREAD

**Issue by joeaudette** [cloudscribe/sample-idserver#2]
# Help Wanted

If you have expertise in React and/or Angular, we would like to have sample clients using those technologies.

Currently we have samples using Google Polymer and a polymer component that I wrote to wrap around the oidc.js, we also have a Xamarin client sample.

The idea is to have working client samples for popular client side technologies that authenticate agains the OPserver project in this solution so that any developer can pull down the repo, run the proejcts and see working sample of authentication using openidconnect via our IdentityServer4 integration for cloudscribe Core.

**Comment by paulvanbladel** [comment]
Joe, 
I have a working version with VueJs.
Is that something you are interested in as well?

**Comment by joeaudette** [comment]
@paulvanbladel certainly! That would be great! Please fork this repo and get a working vue client setup when you have time and then make a pull request when ready

**Comment by joeaudette** [comment]
@paulvanbladel I'm seeing a github alert indicating the vue sample is using a vulnerable version of bootstrap. I tried just updating the version but that completely broke the look of the site so I reverted it.
Could you fix this and make a pull request?

![bootstrap-issue](https://user-images.githubusercontent.com/101627/48430771-13254b80-e73e-11e8-8c34-2d62234a4101.jpg)
